### ISSUE TITLE
Remediation is not working when referencing a whole OVAL file

### ISSUE THREAD

**Issue by abergmann** [OpenSCAP/openscap#1403]
####  Description of Problem:
I've noticed a remediation problem when referencing a whole OVAL file without selecting a definition via a ```name``` value inside a XCCDF rule stanza.
```
<Rule id="xccdf_org.ssgproject.content_rule_test_file_presence_all" selected="false" severity="medium">
  <title>Test file presence all</title>
  <fix system="urn:xccdf:fix:script:sh">touch /tmp/file1; touch /tmp/file2</fix>
  <check multi-check="true" system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
    <check-content-ref href="oval.xml"/>
  </check>
</Rule>
```
When a rule fails openscap is starting the remediation phase, but no script gets executed.
```
#> oscap xccdf eval --remediate --profile standard xccdf.xml 
Title   Test file presence all
Rule    xccdf_org.ssgproject.content_rule_test_file_presence_all
Result  fail

Title   Test file presence all
Rule    xccdf_org.ssgproject.content_rule_test_file_presence_all
Result  fail


 --- Starting Remediation ---
```

#### OpenSCAP Version:
```
#> oscap -V
OpenSCAP command line tool (oscap) 1.3.1
Copyright 2009--2018 Red Hat Inc., Durham, North Carolina.

==== Supported specifications ====
XCCDF Version: 1.2
OVAL Version: 5.11.1
CPE Version: 2.3
CVSS Version: 2.0
CVE Version: 2.0
Asset Identification Version: 1.1
Asset Reporting Format Version: 1.1
CVRF Version: 1.1
```

#### Operating System & Version:
Running on SUSE Linux Enterprise Server 12 SP4.

#### Steps to Reproduce:

Simply follow the steps I've outlined [here](https://gist.github.com/abergmann/5e48b2c0eb7888bc9b0082fed8fae986)

#### Actual Results:
The remediation commands do not get executed.

#### Expected Results:
All commands listed in the ```<fix>...</fix>``` section should be executed when the rule failed before.

**Comment by DominiqueDevinci** [comment]
@abergmann I think it's not a but but a desired behavior. You can find the problem origin in my linked PR (which currently only adds verbose details concerning this case)

**Comment by evgenyz** [comment]
@jan-cerny So, was this bug fixed by #1410 or not?

**Comment by jan-cerny** [comment]
Yes, I tested it and the problem doesn't occur with OpenSCAP 1.3.2.
